health/REFERENCE.md

```diff
@@ -391,7 +391,7 @@ repeat: [off] [warning DURATION] [critical DURATION]
 The only possible value for the `option` line is
```

Comment:
```suggestion
The only possible value for the `options` line is
```

And in the title too https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-option